airdrop/airdrop-distributor/tests/functions/merkle_root.rs

```diff
@@ -0,0 +1,27 @@
+use crate::utils::{
+    airdrop_distributor_abi_calls::{airdrop_constructor, merkle_root},
+    test_helpers::setup,
+};
+
+mod success {
+
+    use super::*;
+
+    #[tokio::test]
+    async fn returns_root() {
+        let (deploy_wallet, _, _, _, asset, claim_time) = setup().await;
+
+        airdrop_constructor(
+            claim_time,
+            &deploy_wallet.airdrop_distributor,
+            [2u8; 32],

```

Comment:
I would consider factoring out this unnamed array into a variable (presumably "merkle root") so that a reader has an easier time understanding this magic structure. I would do this in all files. Having multiple roots to choose from would be fine too. If that is a valid idea then a struct containing them could be returned from the setup